core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -565,6 +565,21 @@ private function addSecurityNode(): NodeDefinition
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('csp')
```

Comment:
Can't we enable this in the root page settings instead? I'd prefer this feature instead of legacy support options :D The `CspReportListener` could ignore reports when the checkbox is not set 🤔

Reply:
We could make the route include the page ID, then I could implement a noise filter that disregards any CSP report if either the page ID is missing or reporting is not enabled in the website root of the page 👍

Reply:
Removed in 13b3c2af15